.github/workflows/check-style.yml

```diff
@@ -1,6 +1,6 @@
 name: Check style
 
-on: [ push ]
+on: [ push, pull_request ]
```

Comment:
I'm not sure if these changes are necessary for this PR; if possible, I'd keep the diff as short as possible

Reply:
They are, tests do not run in forks w/o this change